docs/src/pages/components/modal/modal.md

```diff
@@ -96,4 +96,4 @@ Additionally, you may give a description of your modal with the `aria-describedb
   </Modal>
   ```
 - The [WAI-ARIA authoring practices](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html) can help you set the initial focus on the most relevant element, based on your modal content.
-- A modal window overlys on either the primary window or another modal window. Windows under a modal are **inert**. That is, users cannot interact with content outside an active modal window.
+- A modal window overlays on either the primary window or another modal window. Windows under a modal are **inert**. That is, users cannot interact with content outside an active modal window.
```

Comment:
I have a strange feeling here. Looking at the git history, I'm the original author of this bullet point. However, reading it back, I don't get the meaning 🙃. Should we remove it? cc @eps1lon

Reply:
Ah ok, I get it now, I was confused by the wording. I have added quotes around "modal window".